README.md

```diff
@@ -102,6 +129,8 @@ We provide you with useful function `withSpark`, which accepts everything that m
 
 After work block ends, `spark.stop()` is called automatically.
 
+Do not use this when running the Kotlin Spark API from a Jupyter notebook.
```

Comment:
I think we need a separate paragraph/chapter on jupyter or maybe even example ipynb with best practices

Reply:
sure!

Reply:
yup.. or the wiki?